src/reduce-to-tally.js

```diff
@@ -0,0 +1,19 @@
+export default reducer
+
+/**
+ * Original Source: https://egghead.io/courses/reduce-data-with-javascript
```

Comment:
Normally we're looking for StackOverflow answers in this repo, but I'll let this one slide :)